core/services/pipeline/task.bridge.go

```diff
@@ -91,11 +95,16 @@ func (t *BridgeTask) Run(ctx context.Context, vars Vars, inputs []Result) Result
 		"url", url.String(),
 	)
 
-	responseBytes, elapsed, err := makeHTTPRequest(ctx, "POST", URLParam(url), requestData, allowUnrestrictedNetworkAccess, t.config)
+	responseBytes, headers, elapsed, err := makeHTTPRequest(ctx, "POST", URLParam(url), requestData, allowUnrestrictedNetworkAccess, t.config)
 	if err != nil {
 		return Result{Error: err}
 	}
 
+	// Look for a `pending` flag.
+	if _, ok := headers["X-Chainlink-Pending"]; ok && t.Async == "true" { // TODO: case sensitivity?
```

Comment:
Yeah I think it's wise to be pretty forgiving on task params. At the very least, `strings.TrimSpace`, but lowercasing is probably a good idea too.

Reply:
Looks like golang handles this for us:

```
// HTTP defines that header names are case-insensitive. The
// request parser implements this by using CanonicalHeaderKey,
// making the first character and any characters following a
// hyphen uppercase and the rest lowercase.
```

Reply:
Why support headers? The task sets it as a return value in the json from the bridge, I've not seen this header before